### DISCUSSION TITLE
How to implement an interactive action in pyAEDT

### DISCUSSION THREAD

**Discussion by IkeChoice** [ansys/pyaedt#3370] [Q&A] [answered]
I have been thinking about an implementation of an interactive feature.

For example, I just wanted to interrogate edge id, face id of objects that currently user selected 
Also it would be great if this action can be triggered by a button or bind key callback functions.
Any idea, please?

Also, is it possible to call ACT functionality? if so, then how?
I am searching this topic on the document, but if any one can share with me any information it would be very appreciated.

**Comment by Samuelopez-ansys** [comment, marked as answer]
Hi @IkeChoice ,

You can choose different open source libraries to create an user interface. I have started a new project using Pyside6, you could find more information here:
https://github.com/ansys/pyaedt-toolkit-template

ACT functionalities can not be called. Which one are you interested?

About take the information of the selected object, you can use **app.modeler.selections**  which will give you the selected object names, then you can perform all needed operations.
